usaspending_api/api_docs/api_documentation/federal_obligations.md

```diff
@@ -0,0 +1,67 @@
+## Federal Obligations
+**Route:** `v2/federal_obligations?fiscal_year=[year]&federal_account=[id]`
+
+**Example:** `v2/federal_obligations?fiscal_year=2017&funding_agency_id=1068`
+
+
+**Method:** `GET`
+
+This route sends a request to the backend to retrieve federal obligations.
+
+### Response (JSON) 
+
+```
+{
+    "page_metadata": {
+        "count": 37,
+        "page": 1,
+        "has_next_page": false,
+        "has_previous_page": false,
+        "next": null,
+        "current": "http://localhost:8000/api/v2/federal_obligations/?fiscal_year=2017&funding_agency_id=1068&limit=100&page=1",
+        "previous": null
+    },
+    "results": [
+        {
+            "id": "2497",
+            "agency_name": "Department of Education",
```

Comment:
We have removed `agency_name` from the response object, as it is not needed by the frontend

Reply:
I forgot I did that last week! Nice memory. Fixed